### PR TITLE
Additional information for running app locally

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,5 +21,6 @@ project.lock.json
 .secrets
 appsettings.json
 local.settings.json
+response.json
```
Comment (neilbmclaughlin) [inline .gitignore]: I considered adding to gitignore but started using `| tee ../response.json` and `jq -r '.statusQueryGetUri' ../response.json` as an alternative.

No preference either way but an update to the readme would be required if you decide it is preferable.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-|apim-host-name|
-|apim-subscription|
-|apim-resource-group|
-|apim-api-name|
-|search-hostname|
-|search-api-version|
-|search-api-admin-key|
+* Start the Function app - `WEBSITE_HOSTNAME=localhost:7071 func start`.
+  Setting `WEBSITE_HOSTNAME` prevents an SSL error being reported when running
+  durable functions locally.
```
Comment (neilbmclaughlin) [inline README.md]: ? Add link to context of the problem (https://stackoverflow.com/questions/53812188/cannot-run-azure-durable-function-locally-via-visual-studio-code)

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ for your development platform
 
 * Start the Function app - `WEBSITE_HOSTNAME=localhost:7071 func start`.
   Setting `WEBSITE_HOSTNAME` prevents an SSL error being reported when running
-  durable functions locally.
+  [durable functions locally](https://github.com/Azure/azure-functions-durable-js/issues/28#issuecomment-453250279).
```
Comment (neilbmclaughlin) [inline README.md]: Just looked at the issue in github and it seems it was fixed in 1.1.3 which is now the latest release (as of 4 days ago).

Upgrading durable-functions should make the workaround redundant.

File: .gitignore (hunk 1 above)
Comment (neilbmclaughlin) [reply .gitignore]: Yes but in the parent directory so doesn't flagged in git.

Comment (st3v3nhunt) [reply .gitignore]: Creating the fie within the repo's dir structure is going to avoid unforeseen problems.

File: README.md (hunk 3 above)
Comment (st3v3nhunt) [reply README.md]: Ok. Separate PR for that change?

**Pull request merged**